client/src/components/product_overview/Carousel.jsx

```diff
@@ -3,25 +3,48 @@ import React from 'react';
 import { MdArrowForward, MdArrowBack, MdArrowDropDown } from 'react-icons/md';
 import List from '@material-ui/core/List';
 import ListItem from '@material-ui/core/ListItem';
+// import Zoom from 'react-img-zoom';
 
 class Carousel extends React.Component {
   constructor (props) {
     super(props);
     this.state = {
-      expandButtonClicked: false
     };
   }
 
-  handleFullScreen () {
-    this.setState({ expandButtonClicked: true });
-  }
-
   render () {
     return (
-      <div className="carousel">
+      this.props.expandedView
+        ? <div className="carousel">
+
+        <div className="carousel_container" style={{ backgroundImage: `url(${this.props.currentImage})` }} >
+          {/* {this.props.expandedView ? <Zoom img={`'${this.props.currentImage}'`} zoomScale={2.5} height={200} width={100} transitionTime={0.5} /> : null } */}
+          <div className="icons_container" style={{ maxHeight: 450, overflow: 'auto' }}>
```

Comment:
Consider moving styling into style sheet to avoid potential odd behaviors in the future